Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```diff
@@ -170,3 +170,88 @@
         Slash: 0.5
         Piercing: 0.6
         Heat: 0.5
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterKnightRed
+  name: red knight armour
+  description: A royal knight's favorite garments.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/knightred.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/knightred.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.7
+        Slash: 0.4
+        Piercing: 0.9
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterKnightYellow
+  name: yellow knight armour
+  description: A royal knight's favorite garments.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/knightyellow.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/knightyellow.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.7
+        Slash: 0.4
+        Piercing: 0.9
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterKnightBlue
+  name: blue knight armour
+  description: A royal knight's favorite garments.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/knightblue.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/knightblue.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.7
+        Slash: 0.4
+        Piercing: 0.9
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterKnightGreen
+  name: green knight armour
+  description: A royal knight's favorite garments.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/knightgreen.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/knightgreen.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.7
+        Slash: 0.4
+        Piercing: 0.9
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterTemplar
+  name: templar armour
+  description: Deus Vult Infidel!
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Armor/knighttemplar.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Armor/knighttemplar.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.6
+        Slash: 0.4
+        Piercing: 0.9
```

Comment:
Any reason why this one has .1 less blunt protection? just curious.

Reply:
The cloth on the templar armour softens some blows. Also yes I could split the values between the helmet and armour.